### PR TITLE
Pxz: new package

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: var/spack/repos/builtin/packages/pxz/package.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Pxz(MakefilePackage):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pxz/package.py]: You don't seem to override any methods. How does the package know where to install?

Comment (matthiasdiener) [reply var/spack/repos/builtin/packages/pxz/package.py]: Thanks, should be fixed now.

**Comment by RemiLacroix-IDRIS** [comment]
Note that it seems like this package probably needs to be edited to actually use the lzma dependency from Spack (no `-I` nor `-L` options are set).

On our system that hasn't lzma-dev installed, compilation fails because "lzma.h" cannot be found.

**Comment by adamjstewart** [comment]
> In fact I'm not even sure the dependency should be against the lzma package, I got it to work using the xz package plus setting the search flags for headers and libs.

`lzma` is deprecated, development has continued in the `xz` package which provides the same libraries.

**Pull request merged**